core/procedures.js

```diff
@@ -35,6 +35,16 @@ const {Workspace} = goog.require('Blockly.Workspace');
 goog.require('Blockly.Events.BlockChange');
 
 
+/**
+ * String for use in the "custom" attribute of a category in toolbox XML.
+ * This string indicates that the category should be dynamically populated with
+ * procedure blocks.
+ * @const {string}
+ * @alias Blockly.Procedures.PROCEDURE_CATEGORY_NAME
+ */
```

Comment:
I think it might be worth having a "see also" referring to the `CATEGORY_NAME` in the other two modules (and the same there).

Reply:
Done.